phoenix-core/src/main/java/org/apache/phoenix/hbase/index/Indexer.java

```diff
@@ -311,7 +311,17 @@ public Result preIncrementAfterRowLock(final ObserverContext<RegionCoprocessorEn
           if (!mutations.isEmpty()) {
               Region region = e.getEnvironment().getRegion();
               // Otherwise, submit the mutations directly here
-                region.batchMutate(mutations.toArray(new Mutation[0]));
+              OperationStatus[] batchMutationStatus = region.batchMutate(
```

Comment:
Please also add to IndexRegionObserver, which is the coproc the new global index framework uses.

Reply:
Done